docs/error-codes.md

```diff
@@ -245,3 +245,11 @@ error and warning codes.
 #### `IL2047`: DynamicallyAccessedMemberTypes in DynamicallyAccessedMembersAttribute on <member> don't match overridden <base member>. All overriden members must have the same DynamicallyAccessedMembersAttribute usage.
 
 - All overrides of a virtual method including the base method must have the same DynamicallyAccessedMemberAttribute usage on all it's components (return value, parameters and generic parameters).
+
+#### `IL2048`: Internal attribute RemoveAttributeInstances is being used on member that is not a type
+
+- Internal attribute RemoveAttributeInstances is a special attribute that should only be used on custom attribute types. If referenced in an assembly, module, method, property or event the internal attribute doesn't take any effect
+
+#### `IL2049`: Not recognized internal attribute
```

Comment:
```suggestion
#### `IL2049`: 'RandomName' is not a recognized internal attribute
```

Similar to above, give information which makes it easier to find and fix the problem